gym_sphero/envs/sphero_env.py

```diff
@@ -151,13 +160,13 @@ def step(self, action):
         self._reset_collisions()
         reward_t = self._calc_reward(obs_t)
         self._done = ((self._num_steps + 1 >= self._max_num_steps_in_episode) or (
-            self._stop_episode_at_collision and obs_t[-1] > 0))
+            self._stop_episode_at_collision and obs_t[-1] > 0) or self._location_misaglined)
         debug_info = {}
         if self._done:
             await self._set_color(*_DONE_COLOR)
         else:
             # take action
-            await self._sphero.roll(action[0], action[1])
+            await self._sphero.roll(2 + action[0] * 10, action[1] * 45)
```

Comment:
Consider special casing stop (speed of 0) here. Otherwise the sphero has no choice but to keep moving.
2 is the min speed we want right? maybe something like `0 if action[0] == 0 else max(2, action[0]*10)`